wurst/_handles/primitives/String.wurst

```diff
@@ -207,6 +208,24 @@ public function string.replace(string oldSubstring, string newSubstring) returns
 			i = s.indexOf(oldSubstring, i + newSubstring.length())
 	return s
 
+/** Returns an integer result based on a lexicographic string comparison of strings s1 and s2. */
+public function stringCompare(string s1, string s2) returns int
```

Comment:
It felt a bit weird to write this as `string.compare`, since this function feels more like a standalone function than a class method (`stringCompare("hi", "bye")` vs `"hi".compare("bye")`). Everything else in this file defines class methods for `string` though - up to you @Frotty, lmk and I can change it.